src/Microsoft.Identity.Web/TokenCacheProviders/MeasureDurationResult.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Identity.Web.TokenCacheProviders
+{
+    internal struct MeasureDurationResult
```

Comment:
we aren't using these right now, but going to leave them for if we want to later (copied from MISE).